steps/xcode-build-version/0.0.1/step.yml

```diff
@@ -0,0 +1,66 @@
+title: Xcode Build Version
+summary: |
+  Gets the project's version and build number
+description: |
+  Gets project's version and build number
+website: https://github.com/nodes-ios/bitrise-step-xcode-build-version
+source_code_url: https://github.com/nodes-ios/bitrise-step-xcode-build-version
+support_url: https://github.com/nodes-ios/bitrise-step-xcode-build-version/issues
+published_at: 2020-02-05T13:56:37.566709Z
+source:
+  git: https://github.com/nodes-ios/bitrise-step-xcode-build-version.git
+  commit: b8d43d6c81c4b5a5c32d03bf199f8e304ed88c1e
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- ios
+- macos
+type_tags:
+- utility
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: git
```

Comment:
it seems the step is not using these dependencies, avid listing unused dependencies, as it increases the step run time.